### ISSUE TITLE
[Feature Request] Change the default Prompt

### ISSUE THREAD

**Issue by bgavrilMS** [AzureAD/microsoft-authentication-library-for-dotnet#2086]
**Which Version of MSAL are you using ?**
4.x

**What authentication flow has the issue?**
* Desktop / Mobile
    * [x] Interactive
    * [ ] Integrated Windows Auth
    * [ ] Username Password
    * [ ] Device code flow (browserless)
* Web App
    * [ ] Authorization code
    * [ ] OBO
* Web API
    * [ ] OBO

**Expected behavior**
If user does not specify a prompt, we should probably not send any prompt and let AAD handle things

**Actual behavior**
We send "Select_account"

**Comment by jennyf19** [comment]
@bgavrilMS why not just use noPrompt, which we already have? Is there a customer ask for this? I would think if they don't specify, then we do don't send a prompt, instead of doing "selectAccount".

**Comment by bgavrilMS** [comment]
@jennyf19 - I agree, but it's a bit of a breaking change. 

Today, if we don't specify, we use "select_account". I think we should not send any value, as you say. 

Kusto Explorer has spent a lot of time trying to get this to work and have eventually found that "no_prompt" is a better option than "Select_account"

**Comment by jennyf19** [comment]
@bgavrilMS B2C would also prefer this as well. 

@jmprieur thoughts on this, as far as the customer perspective?

**Comment by henrik-me** [comment]
We should default to not sending any prompt value, I believe that is the latest recommendation from ests

**Comment by jmprieur** [comment]
yes, I agree. "no prompt" provides the best experience and does a select_account when needed. That's what we should use. I'm not even sure that this would break anything (but we'd have to test a bit)

**Comment by bgavrilMS** [comment]
Ok, I added this on the 4.21 release cycle.
